### ISSUE TITLE
[BUG] Table name with multi line comments

### ISSUE THREAD

**Issue by ozum** [cweijan/vscode-mysql#740]
- OS: MacOS
- Database Type: PostgreSQL
- Extension Version: 6.0.7

Hi,

The table name with multi-line comments appears wrong. Please see the screenshot below:

![image](https://user-images.githubusercontent.com/1497060/210128749-1cf535a7-0346-42e4-8ddb-0319ede0aa1a.png)

Here is a DDL to recreate the bug:

```sql
CREATE TABLE "a" (
  "id" UUID DEFAULT gen_random_uuid() NOT NULL
);

CREATE TABLE "b" (
  "id" UUID DEFAULT gen_random_uuid() NOT NULL
);
COMMENT ON TABLE "b" IS 'Some text

Some other text';

CREATE TABLE "c" (
  "id" UUID DEFAULT gen_random_uuid() NOT NULL
);
```

**Comment by cweijan** [comment]
Thanks for the feedback, I will fix it in the next version.

**Comment by cweijan** [comment]
Fix at version 6.0.8.

**Comment by ozum** [comment]
Thanks for the effort. It is fixed.
